packages/ui/src/Card.module.scss

```
@@ -0,0 +1,36 @@
+@use '../styles/constants' as c;
+
+.wrapper {
+  padding: c.$grid * 5;
+  box-shadow: 0 c.$cardShadowSize c.$cardShadowBlur 0 c.$colorShadow;
+  border-radius: c.$borderRadius;
+  background: c.$colorNeutralWhite;
+
+  h3.heading {
```

nit: I don't think we need to specify `h3` here:
```suggestion
  .heading {
```